package.json

```diff
@@ -39,7 +39,8 @@
   "homepage": "https://github.com/webdriverio/wdio-cucumber-framework#readme",
   "dependencies": {
     "babel-runtime": "~6.23.0",
-    "cucumber": "~1.3.1",
+    "cucumber": "lgandecki/cucumber-js#passOptionsToFunctionWrapperNPM",
```

Comment:
I've seen the pending PR here: https://github.com/cucumber/cucumber-js/pull/838
Let's wait for the v1.0.0 release until this is merged any published or maybe release it as beta and use the forked version.